nextjs-ssr/e2e/checkApps.cy.ts

```diff
@@ -27,11 +24,8 @@
                     appName: string
                     host: number
                 }) => {
-            // TODO : Uncomment after fixing the Error: Hydration failed 
-                // let appName = property.host === 3001 ? appsData[0].appName : property.host === 3002 ? appsData[1].appName : appsData[2].appName;
-                // let host = property.host === 3001 ? appsData[0].host : property.host === 3002 ? appsData[1].host : appsData[2].host;
-                const appName = property.host === 3001 ? appsData[0].appName :  appsData[1].appName;
-                const host = property.host === 3001 ? appsData[0].host : appsData[1].host;
+                let appName = property.host === 3001 ? appsData[0].appName : property.host === 3002 ? appsData[1].appName : appsData[2].appName;
```

Comment:
do not like the idea of ternar iinside ternar , it is unreadable. Pls find another way

Reply:
No need at all. Fixed